docs/isp_flashing_guide.md

```diff
@@ -182,6 +182,13 @@ Precompiled `.hex` files are generally not available, but you can compile it you
 
 Note that some boards may have their own specialized build of this bootloader in a separate repository. This will usually be linked to in the board's readme.
 
+### Ubaboot
+
+This bootloader is designed to occupy the bare minimum possible, using less that 512 bytes of flash. It runs on ATmega32U MCUs. It can be flashed with any ISP supported by avrdude.
```

Comment:
32U***4*** i guess, Zadig list also stated "32U", does it really appear with such name?